include/libdnf5-cli/tty.hpp

```diff
@@ -54,6 +54,8 @@ LIBDNF_CLI_API std::ostream & cursor_show(std::ostream & stream);
 
 LIBDNF_CLI_API int get_width();
 LIBDNF_CLI_API bool is_interactive();
+LIBDNF_CLI_API void set_colorized(bool);
+LIBDNF_CLI_API bool is_colorized();
```

Comment:
I'm thinking of names. Actually the function does not return whether the tty is colored. It returns the user's preference of whether or not they want colors based on what they set with `set_colorized`. So on a colored `tty` it may return `false` and vice versa.

But I can't think of a better name... `set_use_colors`, `get_use_colors` ... I don't know

Reply:
And if we keep the convention with `is` we get:
`void coloring_enable(bool)` and `bool is_coloring_enabled()`